src/nft/components/profile/list/ListPage.tsx

```diff
@@ -98,6 +98,12 @@ const FloatingConfirmationBar = styled(Row)`
   transform: translateX(-50%);
   max-width: 1200px;
   z-index: ${Z_INDEX.under_dropdown};
+
+  @media screen and (max-width: ${BREAKPOINTS.sm}px) {
+    width: calc(100% - ${LIST_PAGE_MARGIN_MOBILE * 2}px);
```

Comment:
slightly confused if this needs the isOnV2Page check or not?

Reply:
The `FloatingConfirmationBar` component is only shown when on V2